apps/sap-commerce-cloud/src/index.tsx

```diff
@@ -7,8 +7,8 @@ import '@contentful/forma-36-tokens/dist/css/index.css';
 import './index.css';
 import AppConfig from './AppConfig/AppConfig';
 
-import Field from './components/Field';
-import Dialog from './components/Dialog';
+import Field from './components/Field/Field';
```

Comment:
I moved both of these to their own folders to follow the pattern of the component + css file living together in a folder